arch/risc-v/src/hpm6000/hpm.h

```diff
@@ -0,0 +1,37 @@
+/****************************************************************************
+ * arch/risc-v/src/hpm6000/hpm.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_RISCV_SRC_HPM6750_HPM6750_H
+#define __ARCH_RISCV_SRC_HPM6750_HPM6750_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <sys/types.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#include <arch/irq.h>
+#include "riscv_internal.h"
+#include "chip.h"
+
+#endif /* __ARCH_RISCV_SRC_HPM6750_HPM6750_H */
```

Comment:
```suggestion
#endif /* __ARCH_RISCV_SRC_HPM6000_HPM6750_H */
```
here and other similar places